### PR TITLE
NAS-102736

### PR DESCRIPTION
**Pull request by aervin**
* Adds ID and name props to all action buttons in volume-list.component
* Quick update to changelog parsing so that newlines work and the response is formatted in a readable way.

### CODE REVIEW

File: src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts [hunk 1]
```diff
@@ -340,6 +340,8 @@ export class VolumesListTableConfig implements InputTableConf {
     if (rowData.type === 'zpool') {
 
       actions.push({
+        id: rowData.name,
+        name: 'export_disconnect',
```
Comment (wblock) [inline src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts]: This should match the label precisely, so maybe:
```suggestion
        name: 'Export/Disconnect',
        label: T(rowData.name),
```
For that matter, menu options ought to all be in the global helptext files, but maybe we haven't done that yet.

Comment (aervin) [reply src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts]: Agree the actual button text ought to be in the helptext files, but those will be translated. I'm guessing we'd like the Ids to be static for automation purposes?

**Comment by lola27** [comment]
rebuild

File: src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts [hunk 2]
```diff
@@ -340,6 +340,9 @@ export class VolumesListTableConfig implements InputTableConf {
     if (rowData.type === 'zpool') {
 
       actions.push({
+        id: rowData.name,
+        name: 'Export/Disconnect',
+        label: T(rowData.name),
```
Comment (lola27) [inline src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts]: should remove the duplicate label (line 345)

**Pull request merged**